tests/bluetooth/bt_crypto/src/test_bt_crypto.c

```diff
@@ -8,8 +8,7 @@
 #include <string.h>
 
 #include <zephyr/ztest.h>
-
-#include <bt_crypto.h>
+#include "bt_crypto.h"
```

Comment:
Is this change correct? The convention we’ve used is double quotes when referencing relative to the local path (of the c-file) and angle brackets when relying on include lookup paths.

Reply:
Ah, just saw the resolved discussion with a slightly different rationale